paimon-core/src/main/java/org/apache/paimon/AbstractFileStore.java

```diff
@@ -142,6 +142,11 @@ public CoreOptions options() {
         return options;
     }
 
+    @Override
+    public boolean mergeSchema(RowType rowType, Boolean allowExplicitCast) {
```

Comment:
boolean instead of Boolean?

Reply:
All related call links use `boolean`. (Looks like you're confusing Java and scala?)

Reply:
yep, my fault.